app/mailers/emailer.rb

```diff
@@ -17,27 +17,33 @@ def submission_confirmation(params)
   end
 
   def confirmation_instructions(record, token, opts = {})
-    contact = AccountService.get(record.salesforce_contact_id)
-    @name = if contact.present?
-              "#{contact['firstName']} #{contact['lastName']}"
-            else
-              record.email
-            end
-    super
+    load_salesforce_contact(record)
+    if record.pending_reconfirmation?
```

Comment:
yeah, `pending_reconfirmation` is basically checking if there's an `unconfirmed_email` present. 

However by default devise uses the same email for confirmation and **re**confirmation so this is what allows us to use 2 different emails.